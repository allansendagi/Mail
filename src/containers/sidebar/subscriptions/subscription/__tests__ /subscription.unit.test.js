import React from 'react';
import {shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import {Subscription} from '../subscription.component';


configure({adapter: new Adapter()});
describe('Subscription', () => {
  test('renders empty subscription', () => {
    const wrapper = shallow(
      <Subscription/>
    );
    expect(wrapper).toMatchSnapshot();
  });

  test('renders broadcasting subscription', () => {
    const wrapper = shallow(
      <Subscription broadcasting label='Productioncoder'/>
    );
    expect(wrapper).toMatchSnapshot();
  });

  test('renders non-broadcasting subscription with new videos', () => {
    const wrapper = shallow(
      <Subscription amountNewVideos={4} label='Productioncoder'/>
    );
    expect(wrapper).toMatchSnapshot();
  });
});