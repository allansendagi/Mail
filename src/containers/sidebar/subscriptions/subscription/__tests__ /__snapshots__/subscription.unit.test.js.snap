// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subscription renders broadcasting subscription 1`] = `ShallowWrapper {}`;

exports[`Subscription renders empty subscription 1`] = `ShallowWrapper {}`;

exports[`Subscription renders non-broadcasting subscription with new videos 1`] = `ShallowWrapper {}`;
